css/main.css

```diff
@@ -754,7 +757,69 @@ h2 {
 }
 
 #footer a:hover {
-  color:#1B7B98
+  color:#05cc47
+}
+
+
+/* Added By Aakash
+The Flash Page CSS*/
+
+body, html {
```

Comment:
Always best to put this html and body wide css at top of your css file.
So that it is first thing in view when someone reads your css.